Tests/AWSSDKSwiftCoreTests/SerializersTests.swift

```diff
@@ -11,65 +11,101 @@ import XCTest
-    
+
     struct B: AWSShape {
         public static var members: [AWSShapeMember] = [
             AWSShapeMember(label: "a", required: true, type: .string),
             AWSShapeMember(label: "b", required: false, type: .list),
             AWSShapeMember(label: "c", required: true, type: .list)
         ]
-        
+
         let a = "1"
         let b = [1, 2]
         let c = ["key": "value"]
     }
-    
+
     struct C: AWSShape {
         public static var members: [AWSShapeMember] = [
             AWSShapeMember(label: "value", required: true, type: .string)
         ]
-        
+
         let value = "hello"
     }
-    
+
     struct D: AWSShape {
         public static var members: [AWSShapeMember] = [
             AWSShapeMember(label: "value", required: true, type: .string)
         ]
-        
+
         let value = "world"
     }
-    
+
+    struct E: AWSShape {
+      public static var members: [AWSShapeMember] = [
+          AWSShapeMember(label: "Member", required: true, type: .list),
+      ]
+
+      let Member = ["memberKey": "memberValue", "memberKey2": "memberValue2"]
+    }
+
     struct A: AWSShape {
         public static var members: [AWSShapeMember] = [
             AWSShapeMember(label: "structure", required: true, type: .structure),
             AWSShapeMember(label: "structures", required: false, type: .list),
-            AWSShapeMember(label: "array", required: true, type: .list)
+            AWSShapeMember(label: "array", required: true, type: .list),
+            AWSShapeMember(label: "member", required: true, type: .list)
         ]
-        
+
         let structure = B()
         let dList: [D] = [D()]
         let cList: [C] = [C()]
         let array: [String] = ["foo", "bar"]
+        let structureWithMember = E()
+        let structureWithMembers: [E] = [E(), E()]
     }
-    
+
     func testSerializeToXML() {
         let node = try! AWSShapeEncoder().encodeToXMLNode(A(), attributes: ["A": ["url": "https://example.com"]])
         XCTAssertEqual(node.attributes["url"], "https://example.com")
-        
+
         let xml = XMLNodeSerializer(node: node).serializeToXML()
-        let expected = "<A url=\"https://example.com\"><Structure><A>1</A><B>1</B><B>2</B><C><key>value</key></C></Structure><DList><Value>world</Value></DList><CList><Value>hello</Value></CList><Array>foo</Array><Array>bar</Array></A>"
+        let expected = "<A url=\"https://example.com\"><Structure><A>1</A><B>1</B><B>2</B><C><key>value</key></C></Structure><DList><Value>world</Value></DList><CList><Value>hello</Value></CList><Array>foo</Array><Array>bar</Array><StructureWithMember><Member><memberKey2>memberValue2</memberKey2><memberKey>memberValue</memberKey></Member></StructureWithMember><StructureWithMembers><Member><memberKey2>memberValue2</memberKey2><memberKey>memberValue</memberKey></Member><Member><memberKey2>memberValue2</memberKey2><memberKey>memberValue</memberKey></Member></StructureWithMembers></A>"
```

Comment:
Interestingly, [I found the GO SDK to use a similar looking test](https://github.com/aws/aws-sdk-go/blob/master/models/protocol_tests/output/ec2.json#L161) and uses this syntax: `<member>abc</member>`

Reply:
Yep, and [they're looking for the `member` too](https://github.com/aws/aws-sdk-go/blob/6a946b29774a7b037a96feb5ea6da48b5df35702/private/protocol/xml/xmlutil/unmarshal.go#L139).

Reply:
If you want to, it should be `member` and not `Member`, but that's just a nit.